client/rpc-api/src/state/error.rs

```diff
@@ -41,6 +41,14 @@ pub enum Error {
 		/// Details of the error message.
 		details: String,
 	},
+	/// Provided count exceeds maximum value.
+	#[display(fmt = "count exceeds maximum value. value: {}, max: {}", value, max)]
```

Comment:
Looks good! But please also update below the `From<Error> for rpc::Error` and just use `BASE_ERROR + 2`